brainstormforce/astra
# AST-450 - fix: Shop & single product page column layout issue with grid based CSS

### Description
- https://wp-astra.atlassian.net/browse/AST-450
- https://wp-astra.atlassian.net/browse/AST-406 - Layout issues
- Issue with users which used - `add_filter( 'astra_apply_flex_based_css', '__return_true' );`

### Screenshots
- https://share.getcloudapp.com/WnuYWo68
- https://share.getcloudapp.com/9ZuxKE77
- https://share.bsf.io/OAuPew8w

### Types of changes
- Column layout CSS updated

### How has this been tested?
- Shop grid layout
- Check responsiveness
- Single product - Related Products grid
- Check product grids in every case (like upsells, cross-sells grid etc)
- Verify once with any page builder's product grid

### Checklist:
